docs/io/fits/usage/unfamiliar.rst

```diff
@@ -145,12 +141,13 @@ array in the column. So, for a variable length field of int16, the
 corresponding format spec
 is, e.g.  'PJ(100)'::
 
-    >>> f = fits.open('variable_length_table.fits')
-    >>> f[1].header['tform5']
-    1PI(20)
-    >>> print(f[1].data.field(4)[:3])
```

Comment:
That one didn't work locally so I replaced it. (This is just a not to myself - probably better to discuss this in a new issue).